specs/core/1_shard-data-chains.md

```diff
@@ -383,8 +385,9 @@ def process_shard_attestations(beacon_state: BeaconState, shard_state: ShardStat
     assert bls_verify(bls_aggregate_pubkeys(pubkeys), message, block.attestations, domain)
     # Proposer micro-reward
     proposer_index = get_shard_proposer_index(beacon_state, shard_state.shard, block.slot)
+    assert proposer_index is not None
```

Comment:
The various `assert proposer_index is not None` can be subsumed into a single one `assert` in `get_shard_proposer_index`.